renpy/webloader.py

```diff
@@ -196,8 +196,6 @@ def enqueue(relpath, rtype, data):
 
 
 def process_downloaded_resources():
-    global queue, to_unlink
```

Comment:
This turned out to be wrong for queue.

Reply:
![image](https://user-images.githubusercontent.com/44340603/198727285-e7b6a8f4-724e-48af-b776-0ecf27354c72.png)
Strange, it works out for me, in a template case. Where does the error trigger ?